### PR TITLE
Add requiresafe

### PR DESCRIPTION
**Pull request by mrb**
Yay!

### CODE REVIEW

**Comment by GordonDiggs** [comment]
:+1:

File: config/engines.yml [hunk 1]
```diff
@@ -155,3 +155,9 @@ scss-lint:
   enable_regexps:
   default_ratings_paths:
     - "**.scss"
+requiresafe:
+  image: codeclimate/codeclimate-requiresafe
+  description: Security tool for Node.js dependencies
+  community: true
+  enable_regexps:
+  default_ratings_paths:
```
Comment (pbrisbin) [inline config/engines.yml]: Should these be filled in with something?

Comment (mrb) [inline config/engines.yml]: @pbrisbin This tool only looks at one file, and doesn't impact grades yet, so I think we're fine for now.

Comment (pbrisbin) [inline config/engines.yml]: I'd expect that one file in the `enable_regexps` then, but OK.

**Pull request merged**